src/colmap/controllers/incremental_mapper.h

```diff
@@ -165,21 +165,38 @@ class IncrementalMapperController : public BaseController {
   void TriangulateReconstruction(
       const std::shared_ptr<Reconstruction>& reconstruction);
 
- private:
   bool LoadDatabase();
+
+  // getter functions for python pipelines
+  const std::string GetImagePath() const { return image_path_; }
```

Comment:
The `Get*` prefix is a bit unnecessary, the codebase doesn't have it anywhere else. Maybe remove it for consistency and conciseness? Same for the corresponding Python bindings.

Reply:
The method name GetReconstructionManager and GetDatabaseCache will collapse with the class name without "Get", so here we may need to find a different name. Also, "get_" is used here in Python bindings before: https://github.com/colmap/colmap/blob/main/src/pycolmap/pipeline/sfm.cc#L251. So I would personally prefer to keep it as it is. Or do you prefer to only keep "Get" for "ReconstructionManager" and "DatabaseCache"?

Reply:
Collapsing with the class name is fine, method named are scoped. This is already like this in `Reconstruction`:
https://github.com/colmap/colmap/blob/06a805386afb3db9dc6704c7ba2a80861bb64f7e/src/colmap/scene/reconstruction.h#L85-L87

Reply:
This is done. Thanks